src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -56,6 +61,10 @@ $-z-index--popover: layer.above($-z-index--ui);
   }
 }
 
+.LayerUiAbsolute {
+  position: absolute;
```

Comment:
Shouldn't this also come with `inset: 0`?

[NOTE]
This is the kind of solution I was thinking might make the scroll bars manageable within the `#root` element, not touching `document.body` styles from inside `useEffect()`

Reply:
about that `body` issue. i've checked some other web apps with similar modals that block body scroll... aaand they had `body` element modified with that `overflow` param just like we do
dunno if we find a better solution for it, but let's leave it for a future discussion as it's already a `debt` in there

and I don;'t believe we need `inset` here, works just fine as is